ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,503 @@
+        }
+    }
+
+    /// Set up the background color of the card and update the icon and text color if necessary
+    private func setupBackgroundColor() {
+        var color: UIColor = Constants.defualtBackgroundColor
+
+        switch backgroundStyle {
+        case .appColor:
+            if let window = window {
+                color = Colors.primaryTint40(for: window)
+                primaryLabel.textColor = Colors.primaryShade20(for: window)
+                secondaryLabel.textColor = Colors.primaryShade20(for: window)
+                iconView?.tintColor = Colors.primaryShade20(for: window)
+            }
+        case .neutral:
+            color = Colors.Button.background // TODO: will be updated to the right value once I have the values from design
+        case .custom:
+            color = cardBackgroundColor
+        }
+
+        self.backgroundColor = color
+    }
+
+    private struct Constants {
+        static let defualtBackgroundColor: UIColor = Colors.surfacePrimary
+        static let borderWidth: CGFloat = UIScreen.main.devicePixel
+        static let borderRadius: CGFloat = 8.0
+        static let borderColor = UIColor(light: Colors.gray100, dark: .clear)
+        static let horizontalCardPaddingTopBottom: CGFloat = 6.0
+        static let verticalCardPaddingBottom: CGFloat = 8.0
+        static let verticalCardPaddingTop: CGFloat = 10.0
+        static let cardPaddingTrailing: CGFloat = 16.0
+        static let xSmallPaddingLeading: CGFloat = 12.0
+        static let cardPaddingLeading: CGFloat = 10.0
+        static let horizontalContentSpacing: CGFloat = 12.0
+        static let verticalContentSpacing: CGFloat = 5.0
+        static let titleDefaultNumberOfLines: Int = 2
+        static let subtitleDefaultNumberOfLines: Int = 2
+        static let smallContentSpacing: CGFloat = 2.0
+        static let largeContentSpacing: CGFloat = 6.0
+        static let largeHorizontalPadding: CGFloat = 16.0
+        static let largeVerticalPadding: CGFloat = 14.0
+    }
+
+    private var iconView: UIImageView?
+
+    private var button: UIButton?
+
+    private var imageView: UIImageView?
+
+    private let primaryLabel: UILabel = {
+        let primaryLabel = UILabel()
+        primaryLabel.translatesAutoresizingMaskIntoConstraints = false
+        primaryLabel.font = TextStyle.subhead.font
+        primaryLabel.textColor = Colors.textPrimary
+        primaryLabel.textAlignment = .left
+        return primaryLabel
+    }()
+
+    private let secondaryLabel: UILabel = {
+        let secondaryLabel = UILabel()
+        secondaryLabel.translatesAutoresizingMaskIntoConstraints = false
+        secondaryLabel.font = TextStyle.footnote.font
+        secondaryLabel.textColor = Colors.textSecondary
+        secondaryLabel.textAlignment = .left
+        return secondaryLabel
+    }()
+
+    private var layoutConstraints: [NSLayoutConstraint] = []
+
+    private func setupLayoutConstraints() {
```

Comment:
Based on the logic I see below, it seems like you will need to call setupLayoutConstraints when the text size category changes.

Reply:
Have you tried changing the text size. Do your cards update correctly?